### ISSUE TITLE
node>=v9.9.0, assert should use strict mode

### ISSUE THREAD

**Issue by Aladdin-ADD** [mysticatea/eslint-plugin-node#134]
node>=9.9.0

```js
var assert = require('assert'); // invalid, should report an error
```

```js
var assert = require('assert').strict; // valid
```

**Comment by mysticatea** [comment]
Thank you for this issue.

However, this is intentional.
Node.js community intended to deprecate four methods (`assert.equal`, `assert.deepEqual`, `assert.deepNotEqual`, and `assert.notEqual`) rather than all of the non-strict methods. See [the commit](https://github.com/nodejs/node/commit/a319e90807bbc74b6d0e85ee9bec697acb68ebcb).
